documentation/contributing/code/index.markdown

```diff
@@ -71,6 +71,22 @@ MoveIt uses *one* folder for all headers from all of its modules: ``include/move
 This is rather non-standard for catkin - catkin would prefer to have headers of each ROS package in a separate folder,
 e.g. `include/moveit_core/...`, `include/moveit_ros_planning/...`, etc.
 
+## pre-commit Formatting Checks
+
+In many of our repos we have a [pre-commit](https://pre-commit.com/) check that runs in CI.
+You can run this locally and in many cases it will fix formatting issues for you.
+To install use pip:
```

Comment:
"We use [pre-commit](https://pre-commit.com/) to check the formatting of our files. 
You can use this locally and set it up to run automatically before you commit something.
To install, use pip:"